lib/helpers/is-angle-bracket-component.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function isDynamicComponent(element) {
+  let open = element.tag.charAt(0);
+
+  let maybeLocal = element.tag.split('.').length > 1;
```

Comment:
This is fine for now, but can you create an issue to address actually tracking locals and expose `isLocal` to each plugin?